mobile/pubspec.yaml

```diff
@@ -64,6 +64,7 @@ dev_dependencies:
   flutter_launcher_icons: "^0.9.2"
   flutter_native_splash: ^2.2.16
   isar_generator: *isar_version
+  mockito: ^5.3.2
```

Comment:
Can this be `dev_dependencies` or it has to be in `dependencies` to work?

Reply:
I think `dev_dependencies` is fine since we only use it in unit tests